src/components/dashboard/widget/dashboard-widget.component.ts

```diff
@@ -18,10 +18,24 @@ export class DashboardWidgetComponent implements OnInit, AfterViewInit, OnDestro
     @Input() name: string;
 
     /** Defines the column the widget is placed in */
-    @Input() col: number;
+    @Input() set col(col: number) {
```

Comment:
Since you're adding an accessor anyway, it would be good to [coerce](https://netbasal.com/trust-but-verify-coerce-your-component-inputs-bdb743e8b579) the value from string as well.